packages/terra-switch/README.md

```diff
@@ -0,0 +1,38 @@
+# Terra Switch
+
+[![NPM version](https://badgen.net/npm/v/terra-switch)](https://www.npmjs.org/package/terra-switch)
+[![Build Status](https://badgen.net/travis/cerner/terra-switch)](https://travis-ci.com/cerner/terra-core)
+
+The terra-switch component, like its real-world counterpart, is used to indicate whether a setting is ON or OFF and provide users with instantaneous feedback.
```

Comment:
Could we elaborate on this description? I'm not really sure the intent of the component when reading this. What is it's real-word counterpart? What is the instantaneous feedback the user received?

Reply:
updated [descriptions](https://github.com/cerner/terra-core/pull/2846/commits/8ba4fb42d6a30245f9e12acc308252d47b22d7d2#diff-ca36062bbe0a1bca64938f49baa5c055) for switch component